app/controllers/concerns/api/v1/error_handler.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    module ErrorHandler
+      extend ActiveSupport::Concern
+
+      private
+
+      # Render Error Message in json_api format
+      # :reek:LongParameterList { max_params: 5 }
+      def render_error(detail:, source: nil, meta: nil, status: :unprocessable_entity, code: nil)
+        errors = errors(detail: detail, source: source, meta: meta, code: code)
+
+        render json: { errors: errors }, status: status
+      end
+
+      def errors(detail:, source: nil, meta: nil, code: nil)
```

Comment:
How about naming this `build_errors`? I find that having a method named `errors` rather misleading as just by reading the name I am not sure what this method does 💭 Also since it does not actually render an error, `build_errors` makes sense to me and it's rather consistent with Rails naming.